### ISSUE TITLE
nail down what exactly cpu utilization in zabbix means

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/ops-issues#303]
the `libvirt.cpu` and `system.util.cpu.blahblah`. the confusion comes from the fact that sometimes cpu utilization can be over 100% for multicore systems.

**Comment by naved001** [comment]
I took a test VM that has 4 vCPUs. 

I loaded a single core using `strees-ng` (`stress-ng -c1 -l 100` for 100% load on a single core), and zabbix agent reported 25% utilization (for key `system.cpu.util[,user]`). I then stressed 2 cores and then zabbix reported 50% utilization.

This means for `system.cpu.util.blablabh` it is taking into account the number of cores. For zabbix to report 100% all cores must be loaded a 100 percent.

So when we calculate the aggregate usage across all our openstack computes, zabbix is factoring in the number of cores.

The `libivrt.cpu` metric for openstack/libvirt instances also factors in the number of cores. So a 100% utilization means all cores are being utilized. I did document this here: https://github.com/CCI-MOC/zabbix-libvirt/blob/master/documentation.md#cpu-time
Also, the function here divides the cpu time by core count: https://github.com/CCI-MOC/zabbix-libvirt/blob/master/zabbix_libvirt/libvirt_checks.py#L152-L153


In summary, we won't have utilization over 100%. 

CC: @rob-baron
